src/types/LoginResponse.ts

```diff
@@ -0,0 +1,7 @@
+import { User } from 'types/User';
+
+export type LoginResponse = {
+    status: 'SUCCESS' | 'ERROR';
```

Comment:
You could create an enum and use it also for if block in Sidebar.container.
For now the status can be SUCCESS, ERROR or NOT_FOUND

Reply:
Done